packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -257,6 +257,25 @@ export function isCompatVersionBelowMinVersion(minVersion: string, config: Compa
 	return semver.compare(compatVersion, minReqVersion) < 0;
 }
 
+/**
+ * Returns true if the given compat config is compliant with ODSP's version requirements.
+ * ! If a summarizer's version is too old, ODSP will nack the summaries with "Upgrade to a newer version of the Fluid client packages to summarize".
+ */
+export function isOdspCompatCompliant(config: CompatConfig): boolean {
+	for (const ltsVersion of defaultCompatVersions.ltsVersions) {
+		if (
+			(typeof config.compatVersion === "string" &&
```

Comment:
Why the different logic here v the others? Checking typeof v checking for undefined. 

Also, if they can be consistent, for readability consider writing a local const helper function to do the comparison.